include/core/JamomaSample.h

```diff
@@ -44,8 +44,9 @@ namespace Jamoma {
 	class SampleBundle {
 		using SampleBundleData = std::vector<SampleVector>;
 		
-		SampleBundleData	mChannels;		// each item in this vector represents a channel, which itself is a vector of samples
-		size_t				mFrameCount;	// if we change the number of channels we could loose the framecount in the vectors contained by mChannels, so we cache it here.
+		SampleBundleData	mChannels;		//!< each item in this vector represents a channel, which itself is a vector of samples
+		size_t				mFrameCount;	//!< if we change the number of channels we could loose the framecount in the vectors contained by mChannels, so we cache it here.
+        size_t              mPaddingAmount = 0;     //!< if padding has been added, this value is used to track the number of extra Frames at the beginning and end
```

Comment:
`mPaddingSize` is better and I will make the change.

My only question about raising the default is if the burden of managing the `mPaddingSize` rests with the calling object or should it be handled internally? Do you want the user to have to mess with this everytime?

With a default of zero, the user *knows* that there is a `mPaddingSize` greater than zero because s/he is the one that set it to that value.